src/main/java/racing/domain/RoundResult.java

```diff
@@ -0,0 +1,17 @@
+package racing.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RoundResult {
+
+    private final List<CarResult> results;
```

Comment:
```suggestion
    private final List<CarResult> carRacingResult;
```
의미를 명확하게 표현해보는 것은 어떨까요?
객체지향적인 코드를 읽고 해석할때 객체명을 보고 객체의 역할을 파악합니다. 
다른 객체에 전달하는 메서드명과 변수명을 보고 어떤 메시지를 전달하는지 이해하게 됩니다. 
메서드명과 변수명이 의미를 잘 표현하지 않고 있다면, 코드 가독성이 떨어져 해석하기 힘든 코드가 됩니다.

하나의 스토리를 다른 사람에게 공유하고 현실 세계의 이벤트를 코드로 잘 풀어 설명한다는 관점에서 변수명과 메서드명에 고민을 많이 해주시면 좋을 것 같습니다.